2024-07-18 15:08:20.968172-03:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "create table [*]User (id integer not null, age integer not null, birthdate date, name varchar(255), surname varchar(255), primary key (id))"; se esperaba "identifier"
Syntax error in SQL statement "create table [*]User (id integer not null, age integer not null, birthdate date, name varchar(255), surname varchar(255), primary key (id))"; expected "identifier"; SQL statement:
create table User (id integer not null, age integer not null, birthdate date, name varchar(255), surname varchar(255), primary key (id)) [42001-224]
2024-07-18 15:09:25.516204-03:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "create table [*]User (id integer not null, age integer not null, birthdate date, name varchar(255), surname varchar(255), primary key (id))"; se esperaba "identifier"
Syntax error in SQL statement "create table [*]User (id integer not null, age integer not null, birthdate date, name varchar(255), surname varchar(255), primary key (id))"; expected "identifier"; SQL statement:
create table User (id integer not null, age integer not null, birthdate date, name varchar(255), surname varchar(255), primary key (id)) [42001-224]
2024-07-18 16:45:41.886537-03:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "drop table if exists [*]User cascade "; se esperaba "identifier"
Syntax error in SQL statement "drop table if exists [*]User cascade "; expected "identifier"; SQL statement:
drop table if exists User cascade  [42001-224]
2024-07-18 16:45:41.897042-03:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLSyntaxErrorException: Error de Sintaxis en sentencia SQL "create table [*]User (age integer not null, birthdate date, id integer not null, name varchar(255), surname varchar(255), primary key (id))"; se esperaba "identifier"
Syntax error in SQL statement "create table [*]User (age integer not null, birthdate date, id integer not null, name varchar(255), surname varchar(255), primary key (id))"; expected "identifier"; SQL statement:
create table User (age integer not null, birthdate date, id integer not null, name varchar(255), surname varchar(255), primary key (id)) [42001-224]
2024-07-20 07:11:33.674786-03:00 database: flush
org.h2.message.DbException: Error General : "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]"
General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1529)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]"
General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 12 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	... 7 more
Caused by: java.lang.AssertionError: 28672 != 24576
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 6 more
2024-07-20 07:11:34.143581-03:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos ha sido cerrada
The database has been closed [90098-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.engine.SessionLocal.getTransaction(SessionLocal.java:1610)
	at org.h2.engine.SessionLocal.startStatementWithinTransaction(SessionLocal.java:1631)
	at org.h2.command.Command.executeQuery(Command.java:186)
	at org.h2.jdbc.JdbcPreparedStatement.executeQuery(JdbcPreparedStatement.java:131)
	at io.agroal.pool.wrapper.PreparedStatementWrapper.executeQuery(PreparedStatementWrapper.java:80)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:246)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:167)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:265)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:145)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:19)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:67)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:204)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:211)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:83)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:76)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:65)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$2(ConcreteSqmSelectQueryPlan.java:139)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:382)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:302)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:526)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:423)
	at org.hibernate.query.Query.getResultList(Query.java:120)
	at io.quarkus.hibernate.orm.panache.common.runtime.CommonPanacheQueryImpl.list(CommonPanacheQueryImpl.java:300)
	at io.quarkus.hibernate.orm.panache.runtime.PanacheQueryImpl.list(PanacheQueryImpl.java:149)
	at io.quarkus.hibernate.orm.panache.runtime.JpaOperations.list(JpaOperations.java:24)
	at io.quarkus.hibernate.orm.panache.runtime.JpaOperations.list(JpaOperations.java:10)
	at io.quarkus.hibernate.orm.panache.common.runtime.AbstractJpaOperations.listAll(AbstractJpaOperations.java:305)
	at com.pabloquiroga.essentials.users.UserRepository.listAll(UserRepository.java)
	at com.pabloquiroga.essentials.users.UserRepository_ClientProxy.listAll(Unknown Source)
	at com.pabloquiroga.essentials.users.UserService.getAllUsers(UserService.java:42)
	at com.pabloquiroga.essentials.users.UserService_ClientProxy.getAllUsers(Unknown Source)
	at com.pabloquiroga.essentials.users.UserResource.getAll(UserResource.java:22)
	at com.pabloquiroga.essentials.users.UserResource$quarkusrestinvoker$getAll_ff0bca512259cdc61f4195f154f924d414f41466.invoke(Unknown Source)
	at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:141)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:599)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1529)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	... 1 more
Caused by: java.lang.AssertionError: 28672 != 24576
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 6 more
2024-07-20 07:11:34.145488-03:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos ha sido cerrada
The database has been closed [90098-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.engine.SessionLocal.getTransaction(SessionLocal.java:1610)
	at org.h2.engine.SessionLocal.startStatementWithinTransaction(SessionLocal.java:1631)
	at org.h2.command.Command.executeQuery(Command.java:186)
	at org.h2.jdbc.JdbcConnection.getQueryTimeout(JdbcConnection.java:753)
	at org.h2.jdbc.JdbcStatement.getQueryTimeout(JdbcStatement.java:727)
	at io.agroal.pool.wrapper.StatementWrapper.getQueryTimeout(StatementWrapper.java:180)
	at org.hibernate.resource.jdbc.internal.ResourceRegistryStandardImpl.close(ResourceRegistryStandardImpl.java:196)
	at org.hibernate.resource.jdbc.internal.ResourceRegistryStandardImpl.release(ResourceRegistryStandardImpl.java:108)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.release(DeferredResultSetAccess.java:331)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:259)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:167)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:265)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:145)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:19)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:67)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:204)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:211)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:83)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:76)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:65)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$2(ConcreteSqmSelectQueryPlan.java:139)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:382)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:302)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:526)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:423)
	at org.hibernate.query.Query.getResultList(Query.java:120)
	at io.quarkus.hibernate.orm.panache.common.runtime.CommonPanacheQueryImpl.list(CommonPanacheQueryImpl.java:300)
	at io.quarkus.hibernate.orm.panache.runtime.PanacheQueryImpl.list(PanacheQueryImpl.java:149)
	at io.quarkus.hibernate.orm.panache.runtime.JpaOperations.list(JpaOperations.java:24)
	at io.quarkus.hibernate.orm.panache.runtime.JpaOperations.list(JpaOperations.java:10)
	at io.quarkus.hibernate.orm.panache.common.runtime.AbstractJpaOperations.listAll(AbstractJpaOperations.java:305)
	at com.pabloquiroga.essentials.users.UserRepository.listAll(UserRepository.java)
	at com.pabloquiroga.essentials.users.UserRepository_ClientProxy.listAll(Unknown Source)
	at com.pabloquiroga.essentials.users.UserService.getAllUsers(UserService.java:42)
	at com.pabloquiroga.essentials.users.UserService_ClientProxy.getAllUsers(Unknown Source)
	at com.pabloquiroga.essentials.users.UserResource.getAll(UserResource.java:22)
	at com.pabloquiroga.essentials.users.UserResource$quarkusrestinvoker$getAll_ff0bca512259cdc61f4195f154f924d414f41466.invoke(Unknown Source)
	at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:141)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:599)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1529)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	... 1 more
Caused by: java.lang.AssertionError: 28672 != 24576
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 6 more
2024-07-20 07:11:34.146415-03:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientConnectionException: La base de datos ha sido cerrada
The database has been closed [90098-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:690)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.engine.SessionLocal.getTransaction(SessionLocal.java:1610)
	at org.h2.engine.SessionLocal.startStatementWithinTransaction(SessionLocal.java:1631)
	at org.h2.command.Command.executeQuery(Command.java:186)
	at org.h2.jdbc.JdbcConnection.getQueryTimeout(JdbcConnection.java:753)
	at org.h2.jdbc.JdbcStatement.getQueryTimeout(JdbcStatement.java:727)
	at io.agroal.pool.wrapper.StatementWrapper.getQueryTimeout(StatementWrapper.java:180)
	at org.hibernate.resource.jdbc.internal.ResourceRegistryStandardImpl.close(ResourceRegistryStandardImpl.java:196)
	at org.hibernate.resource.jdbc.internal.ResourceRegistryStandardImpl.release(ResourceRegistryStandardImpl.java:108)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.release(DeferredResultSetAccess.java:331)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.executeQuery(DeferredResultSetAccess.java:259)
	at org.hibernate.sql.results.jdbc.internal.DeferredResultSetAccess.getResultSet(DeferredResultSetAccess.java:167)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.advanceNext(JdbcValuesResultSetImpl.java:265)
	at org.hibernate.sql.results.jdbc.internal.JdbcValuesResultSetImpl.processNext(JdbcValuesResultSetImpl.java:145)
	at org.hibernate.sql.results.jdbc.internal.AbstractJdbcValues.next(AbstractJdbcValues.java:19)
	at org.hibernate.sql.results.internal.RowProcessingStateStandardImpl.next(RowProcessingStateStandardImpl.java:67)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:204)
	at org.hibernate.sql.results.spi.ListResultsConsumer.consume(ListResultsConsumer.java:33)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.doExecuteQuery(JdbcSelectExecutorStandardImpl.java:211)
	at org.hibernate.sql.exec.internal.JdbcSelectExecutorStandardImpl.executeQuery(JdbcSelectExecutorStandardImpl.java:83)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:76)
	at org.hibernate.sql.exec.spi.JdbcSelectExecutor.list(JdbcSelectExecutor.java:65)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.lambda$new$2(ConcreteSqmSelectQueryPlan.java:139)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.withCacheableSqmInterpretation(ConcreteSqmSelectQueryPlan.java:382)
	at org.hibernate.query.sqm.internal.ConcreteSqmSelectQueryPlan.performList(ConcreteSqmSelectQueryPlan.java:302)
	at org.hibernate.query.sqm.internal.QuerySqmImpl.doList(QuerySqmImpl.java:526)
	at org.hibernate.query.spi.AbstractSelectionQuery.list(AbstractSelectionQuery.java:423)
	at org.hibernate.query.Query.getResultList(Query.java:120)
	at io.quarkus.hibernate.orm.panache.common.runtime.CommonPanacheQueryImpl.list(CommonPanacheQueryImpl.java:300)
	at io.quarkus.hibernate.orm.panache.runtime.PanacheQueryImpl.list(PanacheQueryImpl.java:149)
	at io.quarkus.hibernate.orm.panache.runtime.JpaOperations.list(JpaOperations.java:24)
	at io.quarkus.hibernate.orm.panache.runtime.JpaOperations.list(JpaOperations.java:10)
	at io.quarkus.hibernate.orm.panache.common.runtime.AbstractJpaOperations.listAll(AbstractJpaOperations.java:305)
	at com.pabloquiroga.essentials.users.UserRepository.listAll(UserRepository.java)
	at com.pabloquiroga.essentials.users.UserRepository_ClientProxy.listAll(Unknown Source)
	at com.pabloquiroga.essentials.users.UserService.getAllUsers(UserService.java:42)
	at com.pabloquiroga.essentials.users.UserService_ClientProxy.getAllUsers(Unknown Source)
	at com.pabloquiroga.essentials.users.UserResource.getAll(UserResource.java:22)
	at com.pabloquiroga.essentials.users.UserResource$quarkusrestinvoker$getAll_ff0bca512259cdc61f4195f154f924d414f41466.invoke(Unknown Source)
	at org.jboss.resteasy.reactive.server.handlers.InvocationHandler.handle(InvocationHandler.java:29)
	at io.quarkus.resteasy.reactive.server.runtime.QuarkusResteasyReactiveRequestContext.invokeHandler(QuarkusResteasyReactiveRequestContext.java:141)
	at org.jboss.resteasy.reactive.common.core.AbstractResteasyReactiveContext.run(AbstractResteasyReactiveContext.java:147)
	at io.quarkus.vertx.core.runtime.VertxCoreRecorder$14.runWith(VertxCoreRecorder.java:599)
	at org.jboss.threads.EnhancedQueueExecutor$Task.doRunWith(EnhancedQueueExecutor.java:2516)
	at org.jboss.threads.EnhancedQueueExecutor$Task.run(EnhancedQueueExecutor.java:2495)
	at org.jboss.threads.EnhancedQueueExecutor$ThreadBody.run(EnhancedQueueExecutor.java:1521)
	at org.jboss.threads.DelegatingRunnable.run(DelegatingRunnable.java:11)
	at org.jboss.threads.ThreadLocalResettingRunnable.run(ThreadLocalResettingRunnable.java:11)
	at io.netty.util.concurrent.FastThreadLocalRunnable.run(FastThreadLocalRunnable.java:30)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1529)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	... 1 more
Caused by: java.lang.AssertionError: 28672 != 24576
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 6 more
2024-07-20 07:11:34.197825-03:00 jdbc[3]: exception
org.h2.jdbc.JdbcSQLNonTransientException: Error General : "Error General : ""org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]""\000aGeneral error: ""org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]"" [50000-224]"
General error: "Error General : ""org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]""\000aGeneral error: ""org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]"" [50000-224]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.engine.Database.throwLastBackgroundException(Database.java:1817)
	at org.h2.engine.SessionLocal.close(SessionLocal.java:890)
	at org.h2.jdbc.JdbcConnection.close(JdbcConnection.java:391)
	at io.agroal.pool.util.XAConnectionAdaptor.close(XAConnectionAdaptor.java:33)
	at io.agroal.pool.ConnectionHandler.closeConnection(ConnectionHandler.java:183)
	at io.agroal.pool.ConnectionPool$DestroyConnectionTask.run(ConnectionPool.java:834)
	at io.agroal.pool.ConnectionPool.close(ConnectionPool.java:200)
	at io.agroal.pool.DataSource.close(DataSource.java:79)
	at io.quarkus.agroal.runtime.DataSources.stop(DataSources.java:449)
	at io.quarkus.agroal.runtime.DataSources_Bean.doDestroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.agroal.runtime.DataSources_Bean.destroy(Unknown Source)
	at io.quarkus.arc.impl.AbstractInstanceHandle.destroyInternal(AbstractInstanceHandle.java:78)
	at io.quarkus.arc.impl.ContextInstanceHandleImpl.destroy(ContextInstanceHandleImpl.java:26)
	at io.quarkus.arc.impl.AbstractSharedContext.destroy(AbstractSharedContext.java:102)
	at io.quarkus.arc.impl.ArcContainerImpl.shutdown(ArcContainerImpl.java:465)
	at io.quarkus.arc.Arc.shutdown(Arc.java:65)
	at io.quarkus.arc.runtime.ArcRecorder$1.run(ArcRecorder.java:53)
	at io.quarkus.runtime.StartupContext.runAllAndClear(StartupContext.java:87)
	at io.quarkus.runtime.StartupContext.close(StartupContext.java:79)
	at io.quarkus.runner.ApplicationImpl.doStop(Unknown Source)
	at io.quarkus.runtime.Application.stop(Application.java:208)
	at io.quarkus.runtime.Application.stop(Application.java:155)
	at io.quarkus.runtime.Application.close(Application.java:137)
	at io.quarkus.runner.bootstrap.StartupActionImpl$5.close(StartupActionImpl.java:296)
	at io.quarkus.runner.bootstrap.RunningQuarkusApplicationImpl.close(RunningQuarkusApplicationImpl.java:35)
	at io.quarkus.test.junit.QuarkusTestExtension$4.close(QuarkusTestExtension.java:285)
	at io.quarkus.test.junit.QuarkusTestExtension$ExtensionState.doClose(QuarkusTestExtension.java:1236)
	at io.quarkus.test.junit.QuarkusTestExtensionState.close(QuarkusTestExtensionState.java:42)
	at org.junit.jupiter.engine.descriptor.AbstractExtensionContext.lambda$static$0(AbstractExtensionContext.java:45)
	at org.junit.platform.engine.support.store.NamespacedHierarchicalStore$EvaluatedValue.close(NamespacedHierarchicalStore.java:333)
	at org.junit.platform.engine.support.store.NamespacedHierarchicalStore$EvaluatedValue.access$800(NamespacedHierarchicalStore.java:317)
	at org.junit.platform.engine.support.store.NamespacedHierarchicalStore.lambda$close$3(NamespacedHierarchicalStore.java:98)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.store.NamespacedHierarchicalStore.lambda$close$4(NamespacedHierarchicalStore.java:98)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.accept(ForEachOps.java:184)
	at java.base/java.util.ArrayList.forEach(ArrayList.java:1596)
	at java.base/java.util.stream.SortedOps$RefSortingSink.end(SortedOps.java:395)
	at java.base/java.util.stream.Sink$ChainedReference.end(Sink.java:261)
	at java.base/java.util.stream.Sink$ChainedReference.end(Sink.java:261)
	at java.base/java.util.stream.AbstractPipeline.copyInto(AbstractPipeline.java:510)
	at java.base/java.util.stream.AbstractPipeline.wrapAndCopyInto(AbstractPipeline.java:499)
	at java.base/java.util.stream.ForEachOps$ForEachOp.evaluateSequential(ForEachOps.java:151)
	at java.base/java.util.stream.ForEachOps$ForEachOp$OfRef.evaluateSequential(ForEachOps.java:174)
	at java.base/java.util.stream.AbstractPipeline.evaluate(AbstractPipeline.java:234)
	at java.base/java.util.stream.ReferencePipeline.forEach(ReferencePipeline.java:596)
	at org.junit.platform.engine.support.store.NamespacedHierarchicalStore.close(NamespacedHierarchicalStore.java:98)
	at org.junit.jupiter.engine.descriptor.AbstractExtensionContext.close(AbstractExtensionContext.java:87)
	at org.junit.jupiter.engine.execution.JupiterEngineExecutionContext.close(JupiterEngineExecutionContext.java:53)
	at org.junit.jupiter.engine.descriptor.JupiterEngineDescriptor.cleanUp(JupiterEngineDescriptor.java:70)
	at org.junit.jupiter.engine.descriptor.JupiterEngineDescriptor.cleanUp(JupiterEngineDescriptor.java:31)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.lambda$cleanUp$10(NodeTestTask.java:167)
	at org.junit.platform.engine.support.hierarchical.ThrowableCollector.execute(ThrowableCollector.java:73)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.cleanUp(NodeTestTask.java:167)
	at org.junit.platform.engine.support.hierarchical.NodeTestTask.execute(NodeTestTask.java:98)
	at org.junit.platform.engine.support.hierarchical.SameThreadHierarchicalTestExecutorService.submit(SameThreadHierarchicalTestExecutorService.java:35)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestExecutor.execute(HierarchicalTestExecutor.java:57)
	at org.junit.platform.engine.support.hierarchical.HierarchicalTestEngine.execute(HierarchicalTestEngine.java:54)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:198)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:169)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:93)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.lambda$execute$0(EngineExecutionOrchestrator.java:58)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.withInterceptedStreams(EngineExecutionOrchestrator.java:141)
	at org.junit.platform.launcher.core.EngineExecutionOrchestrator.execute(EngineExecutionOrchestrator.java:57)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:103)
	at org.junit.platform.launcher.core.DefaultLauncher.execute(DefaultLauncher.java:85)
	at org.junit.platform.launcher.core.DelegatingLauncher.execute(DelegatingLauncher.java:47)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor$CollectAllTestClassesExecutor.processAllTestClasses(JUnitPlatformTestClassProcessor.java:119)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor$CollectAllTestClassesExecutor.access$000(JUnitPlatformTestClassProcessor.java:94)
	at org.gradle.api.internal.tasks.testing.junitplatform.JUnitPlatformTestClassProcessor.stop(JUnitPlatformTestClassProcessor.java:89)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.stop(SuiteTestClassProcessor.java:62)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	at java.base/java.lang.reflect.Method.invoke(Method.java:580)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:36)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:33)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:94)
	at jdk.proxy1/jdk.proxy1.$Proxy2.stop(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker$3.run(TestWorker.java:193)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.executeAndMaintainThreadName(TestWorker.java:129)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:100)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.execute(TestWorker.java:60)
	at org.gradle.process.internal.worker.child.ActionExecutionWorker.execute(ActionExecutionWorker.java:56)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:119)
	at org.gradle.process.internal.worker.child.SystemApplicationClassLoaderWorker.call(SystemApplicationClassLoaderWorker.java:66)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.run(GradleWorkerMain.java:69)
	at worker.org.gradle.process.internal.worker.GradleWorkerMain.main(GradleWorkerMain.java:74)
Caused by: org.h2.message.DbException: Error General : "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]"
General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.get(DbException.java:212)
	at org.h2.message.DbException.convert(DbException.java:407)
	at org.h2.mvstore.db.Store.lambda$new$0(Store.java:122)
	at org.h2.mvstore.MVStore.handleException(MVStore.java:1546)
	at org.h2.mvstore.MVStore.panic(MVStore.java:371)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1529)
	at org.h2.mvstore.FileStore.lambda$serializeAndStore$3(FileStore.java:1432)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1583)
Caused by: org.h2.jdbc.JdbcSQLNonTransientException: Error General : "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]"
General error: "org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]" [50000-224]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:566)
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:489)
	... 12 more
Caused by: org.h2.mvstore.MVStoreException: java.lang.AssertionError: 28672 != 24576 [2.2.224/3]
	at org.h2.mvstore.DataUtils.newMVStoreException(DataUtils.java:996)
	... 7 more
Caused by: java.lang.AssertionError: 28672 != 24576
	at org.h2.mvstore.RandomAccessStore.shrinkStoreIfPossible(RandomAccessStore.java:679)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:377)
	at org.h2.mvstore.RandomAccessStore.writeChunk(RandomAccessStore.java:28)
	at org.h2.mvstore.FileStore.storeBuffer(FileStore.java:1524)
	... 6 more
